src/test/java/org/zalando/typemapper/postgres/PgSerializerTest.java

```diff
@@ -83,19 +79,23 @@ public static Collection<Object[]> generateData() throws SQLException {
                                 ARRAY("first element", "second \"quoted\" element").asJdbcArray("text"),
                                 "{\"first element\",\"second \\\"quoted\\\" element\"}"
                         },
-                        {ROW(1, 2), "(1,2)"},
-                        {ROW("a", "b{x}"), "(a,b{x})"},
-                        {ROW("a", "b(x)"), "(a,\"b(x)\")"},
-                        {ROW(1, 2, ARRAY("a", "b")), "(1,2,\"{a,b}\")"},
-                        {ROW("a", "b", new int[]{1, 2, 3, 4}), "(a,b,\"{1,2,3,4}\")"},
-                        {ROW("a", null, ARRAY(ROW(1), ROW(2), null)), "(a,,\"{(1),(2),NULL}\")"},
+                        {row(1, 2), "(1,2)"},
```

Comment:
I don't think this is the correct fix. You've implemented separate version of `row` while tests and the library uses `PgRow.row` which is used in different parts of the library including mapper for procedure parameters (`OtherStoredProcedureParameter`). This fix makes test work, but in production the behaviour could be different. 

I suggest to apply a fix to `PgRow` and use the same `row` implementation for tests and the library itself.

Reply:
done